app/Services/Campaign/ExportService.php

```diff
@@ -48,11 +53,25 @@ class ExportService
 
     protected int $currentElements;
 
+    protected EntityExportService $service;
+
+    public function __construct(EntityExportService $service)
```

Comment:
Can be simplified with `__construct(protected EntityExportService $entityExportService)`